### DISCUSSION TITLE
SmartAutoClicker-2.4.2

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#373] [Announcements]
## Bug fixes

* Fix an issue preventing to capture conditions/detection conditions in landscape mode on Android 14. (see https://github.com/Nain57/Smart-AutoClicker/issues/338)
* Fix copy button in dumb actions brief
* Fix default scenario quality for tutorials

**Full Changelog**: https://github.com/Nain57/Smart-AutoClicker/compare/2.4.1...2.4.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/2.4.2'>SmartAutoClicker-2.4.2</a>.</em>

**Comment by iamLiquidX** [comment]
How much of wait till the update appears in play store?
Fabulous work , Android 14 bug was annoying. Always had to force restart the device

**Reply by Nain57** [reply]
Update is generated and pushed to the playstore automatically when I write the release note. 
The only delay between the actual PlayStore deployment and the release build is the Google review. If everything is OK, it usually takes 2 to 4 days to become available.

**Comment by AndreySeoUa** [comment]
Good afternoon.  It would be great if you could expand the functionality like in the Ankulua application.  for example, making your own execution conditions, clicking, for example, to the left of the found image, pressing buttons, writing code manually.

**Comment by FnugDev** [comment]
Make it for jailbreaked ios devices pls

**Reply by Nain57** [reply]
Doing it for iOS will be a completely different app, apis and limitations are completely different  (I don't even know if  it is possible with a jailbreak due to OS capabilities). Given the limited user base (jailbreaks are less and less common) and the risks, it will not be done.

**Reply by c-sanchez** [reply]
I think it is more practical for you to buy an Android Smartphone 😂

I would also like to have Smart-AutoClicker for desktop (Windows / Linux / Mac), however as with iOS, they are completely different systems.

**Reply by Nain57** [reply]
For Windows, you can try [AutoHotkey](https://github.com/AutoHotkey/AutoHotkey/tree/v2.0). 
Last time i checked, it was able to do the same things as Smart AutoClicker (and even more), but it was less user friendly

**Reply by c-sanchez** [reply]
Yeah @Nain57, AutoHotkey is great, however you would basically need to write code.
Luckily though there is [Pulover's Macro Creator](https://www.macrocreator.com/), which can be seen as a graphical interface to create macros with AutoHotkey :)

**Comment by aeventolero** [comment]
Hello is it possible to have a functionality where in you can put a random interval for the wait function ? Like for example i want the wait time to be randomize between 2 mins to 5 mins ?

**Comment by FMorgas** [comment]
Hello. Thank you for this amazing app.
I miss the functionality of interact with the main button such as "Press back" or "Press Home" or "Open Apps History".
There are any way using intent to for instance doing a press back button?
